src/dotnet-uninstall/Shared/Utils/Regexes.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Text.RegularExpressions;
+
+namespace Microsoft.DotNet.Tools.Uninstall.Shared.Utils
+{
+    internal static class Regexes
```

Comment:
This makes me want to run for the hills.

Reply:
I agree. This class is kind of messy. But if we don't do this then we would have duplicate regexes spread in the entire project and updating a regex requires updating those copies, which I think is very hard to maintain.